contracts/interfaces/IAllocationStrategy.sol

```diff
@@ -15,8 +15,11 @@ interface IAllocationStrategy {
         Rejected
     }
 
+    // Initializer
+    function initialize(bytes memory _data) external;
+
     // call to allo() to get identity for pool, then to registry() to get metadata
-    function getOwnerIdentity() external view returns (string memory);
+    function getOwnerIdentity() external view returns (bytes32);
```

Comment:
@zobront updating this to return the identityId of the pool creator

Reply:
@zobront for reference we moved two functions to `IStrategy.sol`

```javascript
interface IStrategy {
    /**
     * STORAGE (with public getters)
     *     bool initialized;
     *     bytes32 identityId;
     *     uint256 poolId;
     *     address allo;
     */

    // initialize the strategy with the poolId and allo address
    // set initialized to true and ensure it can't be called again
    // check if identityId passed, is same as the identityId set during deployment
    // if identityId is not set during deployment, then set it (for clones)
    function initialize(bytes32 _identityId, uint256 _poolId, address _allo, bytes memory _data) external;

    // call to allo() to return identity for pool
    function getOwnerIdentity() external view returns (bytes32);
}
```